modules/creating-a-custom-ingress-controller.adoc

```diff
@@ -0,0 +1,66 @@
+// Module included in the following assemblies:
+//
+// *ingress-controller-dnsmgt.adoc
+
+:_content-type: PROCEDURE
+[id="creating-a-custom-ingress-controller_{context}"]
+= Creating a custom Ingress Controller
+
+As a cluster administrator, you can create a custom Ingress Controller to manage DNS record lifecycle.
+
+.Prerequisites
+
+* Install the OpenShift CLI (`oc`).
+* Log in as a user with `cluster-admin` privileges.
+* DNS record created.
+
+.Procedure
+
+. Create an `IngressController` custom resource to set `dnsManagementPolicy`:
+
++
+[source,terminal]
+----
+oc -n openshift-ingress-operator patch ingresscontrollers/<name> --type=merge --patch='{"spec":{"endpointPublishingStrategy":{"loadBalancer":{"dnsManagementPolicy":"Unmanaged"}}}}'
+----
+
++
+[NOTE]
+====
+Once you update `.spec.endpointPublishingStrategy.loadBalancer.dnsManagementPolicy` field to `Unmanaged`, the existing DNS record must denote the same using the `dnsManagementPolicy` field. You can either retain or delete the DNS record in the cloud provider.
+If the domain on the Ingress Controller does not match with `dnsConfig.Spec.BaseDomain` , the policy is automatically set to `Unmanaged`.
+====
+
++
+.Sample Ingress Controller configuration
++
+[source,yaml]
```

Comment:
This should be the main content for this section, so we can drop `Sample Ingress Controller configuration` and just keep the body.

Reply:
ok noted